DNN Platform/Modules/DnnExportImport/Components/Services/PagesExportService.cs

```diff
@@ -1876,12 +1888,28 @@ private void SetPartialImportSettings(ExportTab exportedTab, TabInfo localTab)
             }
         }
 
-        private void AddIntoPartialImportedTabs(ExportTab tabToExport)
+        private void CheckForPartialImportedTabs(ExportTab tabToExport)
         {
             var exportTabParentId = tabToExport.ParentId.GetValueOrDefault(Null.NullInteger);
-            if (!_partialImportedTabs.ContainsKey(tabToExport.LocalizedVersionGuid) && exportTabParentId != -1)
+
+            if (exportTabParentId == -1)
+            {
+                if (int.TryParse(tabToExport.Url, out exportTabParentId))
```

Comment:
This is a bit weird. Why parse `Url` into string?

Reply:
It is by designed when URL property of page is type of number then that number(tabId) represent the parent Id of the page.